### DISCUSSION TITLE
ESP32-S3 Secure boot and tinyUF2

### DISCUSSION THREAD

**Discussion by rtborg** [adafruit/tinyuf2#324] [Q&A] [answered]
Is it possible to use secure boot and tinyUF2 on ESP32-S3? 

If it is, will the binaries flashed through tinyUF2 still undergo signature checks?

**Comment by hathach** [comment, marked as answer]
it is possible, uf2 is just a mean to provide binary. Though somebody need to implement this feauture, PR is welcome.
